src/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { render } from '@testing-library/react';
+import App from '../App.js';
```

Comment:
```suggestion
import App from '../app.js';
```

Reply:
Changing the case of the App.js file throws an eslint error

Reply:
Changing the case of the App.js file throws an eslint error.
Something about not being the convention with react, where all component names must start with an uppercase letter

Reply:
Then in that case rename the file under `src/app.js` to `src/App.js`

Reply:
Done

Reply:
Finally went with the lowercase naming convention for the app file